query-engine/connector-test-kit/src/test/scala/writes/uniquesAndNodeSelectors/relation_uniques/CompoundUniqueRelationFieldSpec.scala

```diff
@@ -14,245 +14,6 @@ import util._
 //
 // 2) Checks basic cursor functionality.
 class CompoundUniqueRelationFieldSpec extends FlatSpec with Matchers with ApiSpecBase {
-  // Mutations in this test:
-  //  create         | root   | checked
-  //  update         | root   | checked
-  //  delete         | root   | not possible (1!:1!)
-  //  upsert         | root   | checked
-  //  updateMany     | root   | unnecessary
-  //  deleteMany     | root   | unnecessary
-  //  nested create  | create | checked
-  //  nested update  | update | checked
-  //  nested connect | -      | not possible (1!:1!)
-  //  nested disconn | -      | not possible (1!:1!)
-  //  nested delete  | -      | not possible (1!:1!)
-  //  nested set     | -      | not possible (1!:1!)
-  //  nested upsert  | -      | not possible (1!:1!)
-  //  nested deleteM | -      | not possible (1!:1!)
-  //  nested updateM | -      | not possible (1!:1!)
-  "Using a compound unique that includes a 1!:1! single-field relation" should "work" in {
```

Comment:
We should probably bring this one back as it is not about 1:1 relations.

Reply:
There are other test cases in that file that test the compound unique behavior but with 1:1!, so I think its fine that this one is gone.